package.json

```diff
@@ -19,10 +19,7 @@
 		"example-test": "cd ./tram-one-example && npm test",
 		"example-test-ci": "cd ./tram-one-example && npx jest",
 		"ci-run": "node tests/circleci-run.js",
-		"postpublish:build": "rm -rf tram-one-example && node generator.js tram-one-example --no-install --no-commit --url tram-one-express",
-		"postpublish:deploy-repo": "gh-pages -d tram-one-example -b tram-one-example",
-		"postpublish:deploy-app": "cd ./tram-one-example && npm ci && npm run deploy",
-		"postpublish": "npm run postpublish:build && npm run postpublish:deploy-repo && npm run postpublish:deploy-app"
```

Comment:
postpublish steps no longer required, will all happen automatically on build (and generate a `master-tram-one-example` branch